public/send-results.js

```diff
@@ -0,0 +1,26 @@
+/* global: location, sessionStorage */
+
+(function() {
+  var path = location.pathname.slice(1)
+
+  if (path === "login") {
+    var submitSurveyButton = document.getElementById('submit-survey-button')
+    var clinicianEmailInput = document.getElementById('clinician-email-input')
+    submitSurveyButton.addEventListener('click', function() {
+      var storeClinicianEmail = sessionStorage.setItem('clinician-email', clinicianEmailInput.value)
+    })
+  }
+
+  if (path === "results") {
+    var sendResultsButton = document.getElementById('send-results-button')
+    var getSessionStorage = window.sessionStorage
+    var getClinicianEmail = getSessionStorage['clinician-email']
```

Comment:
Hmm, can you be a bit more consistent in the way using session storage.

Ie https://github.com/lucyrose93/MakaTalk/blob/master/public/confirm-survey.js#L4

Also I am not sure using session storage this way is in the session storage spec